kedro-datasets/setup.py

```diff
@@ -58,6 +58,7 @@ def _collect_requirements(requires):
     "pandas.ParquetDataSet": [PANDAS, "pyarrow>=6.0"],
     "pandas.SQLTableDataSet": [PANDAS, "SQLAlchemy~=1.2"],
     "pandas.SQLQueryDataSet": [PANDAS, "SQLAlchemy~=1.2"],
+    # TODO: Add the mssql dataset requirements here. Or maybe not needed?
```

Comment:
Unresolved TODO - you'll need to add `pyodbc` here and any other dependencies

Reply:
What is the best practice you use here to make it optional? :thinking:

Reply:
It looks like `pyodbc` is a relatively small package so it should be fine to include for all cases 👍